opentracing-example/src/main/scala/zio/telemetry/opentracing/example/http/StatusesService.scala

```diff
@@ -16,28 +17,29 @@ import scala.jdk.CollectionConverters._
 
 object StatusesService {
   def statuses(backendUri: Uri, service: ZLayer[Clock, Throwable, Clock with OpenTracing]): HttpApp[Clock, Throwable] = {
-    HttpApp.collectM { case Method.GET -> Root / "statuses" =>
-      val zio =
-        for {
-          _       <- OpenTracing.tag(Tags.SPAN_KIND.getKey, Tags.SPAN_KIND_CLIENT)
-          _       <- OpenTracing.tag(Tags.HTTP_METHOD.getKey, "GET")
-          _       <- OpenTracing.setBaggageItem("proxy-baggage-item-key", "proxy-baggage-item-value")
-          buffer  <- UIO.succeed(new TextMapAdapter(mutable.Map.empty[String, String].asJava))
-          _       <- OpenTracing.inject(HttpHeadersFormat, buffer)
-          headers <- extractHeaders(buffer)
-          up       = Status.up("proxy")
-          res     <- Client
-                       .status(backendUri.withPath("status"), headers)
-                       .map(_.body)
-                       .flatMap {
-                         case Right(s) => ZIO.succeed(Response.jsonString(Statuses(List(s, up)).toJson))
-                         case _        => ZIO.succeed(Response.jsonString(Statuses(List(Status.down("backend"), up)).toJson))
-                       }
-        } yield res
+    HttpApp.collectM {
+      case Method.GET -> Root / "statuses" =>
+        val zio =
+          for {
+            _       <- OpenTracing.tag(Tags.SPAN_KIND.getKey, Tags.SPAN_KIND_CLIENT)
+            _       <- OpenTracing.tag(Tags.HTTP_METHOD.getKey, GET.method)
+            _       <- OpenTracing.setBaggageItem("proxy-baggage-item-key", "proxy-baggage-item-value")
+            buffer  <- UIO.succeed(new TextMapAdapter(mutable.Map.empty[String, String].asJava))
+            _       <- OpenTracing.inject(HttpHeadersFormat, buffer)
+            headers <- extractHeaders(buffer)
+            up       = Status.up("proxy")
+            res     <- Client
+                         .status(backendUri.withPath("status"), headers)
+                         .map(_.body)
+                         .flatMap {
+                           case Right(s) => ZIO.succeed(Response.jsonString(Statuses(List(s, up)).toJson))
+                           case _        => ZIO.succeed(Response.jsonString(Statuses(List(Status.down("backend"), up)).toJson))
+                         }
```

Comment:
`jsonString` is pure, therefore you can simplify:

```suggestion
                         .map { res =>
                           val status = res.body.getOrElse(Status.down("backend"))
                           val statuses =  Statuses(List(status, up))
                           Response.jsonString(statuses.toJson)
                         }
```